### DISCUSSION TITLE
Question about extra maintainers

### DISCUSSION THREAD

**Discussion by dertseha** [inkyblackness/imgui-go#161] [Q&A] [answered]
Hi all!

It may come in ebb and flow, yet I found myself less often working on `imgui-go` in the past weeks and months, and then also responding to PRs with quite some delay.

My question to you, both users and contributors: How do you see this? By extend: Would you see benefit in having more than one maintainer of this repo? Who would that be?

**Comment by JetSetIlly** [comment]
I don't perceive delays as being problematic. We're fortunate that the Go module system allows us to temporarily point a module to a forked repository. Of course it's best if changes are folded back into the main repository but it certainly isn't impact my releases.

The question really, is whether _you_ see the benefit of having more than one maintainer. That person needs to keep the same policies as you however. That is, no experimental branches, no third-party widgets being added.

**Comment by dertseha** [comment, marked as answer]
This question is implicitly answered through the planned discontinuation. See #192 .
